pages/common/convert.md

```diff
@@ -17,3 +17,7 @@
 - Horizontally append images:
 
 `convert {{image1.png}} {{image2.png}} {{image3.png}} +append {{image123.png}}`
+
+- Create gif from series of images:
+
+`convert {{image1.png}} {{image2.png}} {{image3.png}} -delay 100 {{animation.gif}}`
```

Comment:
100 in tokens.

Reply:
The other examples set a specific value for the purpose of the example. I was following that format.

Reply:
Yes, we prefer specific values in examples. I just want the value to be in tokens.